src/pages/developing/angular-tutorial/wrapping-up.mdx

```diff
@@ -33,7 +33,7 @@ available to anybody who completes the tutorial, not just IBM employees.
 
 ![](/images/carbon-badge.png)
 
-<Caption>IBM Digital Badge</Caption>
+<Caption>IBM Digital badge</Caption>
```

Comment:
Why update casing here but not consistently across the entire page? There are other uses of "IBM Digital Badge".

Reply:
@janchild Any thoughts here?

Reply:
@andreancardona @mattrosno I'll go through the file and make sure it's consistent.